docs/source/using_the_ve/virtual_ecosystem_in_static_mode.md

```diff
@@ -0,0 +1,440 @@
+
+In practice this means:
+
+- **Microclimate is fixed**: e.g., temperature and relative humidity stay constant
+- **Plants are static**: e.g. water uptake, transpiration, and interception do not vary
+- **Soil microbes, litter, and animals do not feedback** into hydrology
+- **Only hydrology processes evolve over time** in response to precipitation
+
+Finally, we compare their outputs to the default setup (`ve_example`) where all
+components run dynamically.
+
+```{note}
+This guide focuses on how to technically configure and run static mode, not on
+interpreting the experimental results. This enables you to adapt the setup to suit your
+own analysis goals.
+
+The example uses `%%bash` for demonstration on Linux, but it is compatible with all
+operating systems.
+```
+
+### 1: Run `ve_example` as a baseline
+
+Before setting up static experiments, make sure you can run the model successfully.
+If you haven’t yet installed and executed the example, follow the
+[example instructions](./virtual_ecosystem_in_use.md) to familiarise yourself with the
+setup.
+
+```{code-cell} ipython3
+%%bash
+# Remove any existing VE data directory in the /tmp/ directory
+if [ -d /tmp/ve_example ]; then
+  rm -r /tmp/ve_example
+fi
+```
+
+```{code-cell} ipython3
+%%bash
+# Install the example data directory from the Virtual Ecosystem package
+ve_run --install-example /tmp/
+```
+
+```{code-cell} ipython3
+:tags: [hide-output]
+
+%%bash
+# Run the example
+ve_run /tmp/ve_example/config \
+  --outpath /tmp/ve_example/out/ \
+  --logfile /tmp/ve_example/out/ve_example.log
+```
+
+### 2: HydroDefault experiment
+
+#### Configure static components
+
+Once the baseline run is complete, you can set up the experiment with default
+'hydrology-only' (no changes to parameters or input data):
+
+- **This is essential**: Create a new output folder: `/ve_example/HydroDefault_out/`.
+  The Virtual Ecosystem output files have fixed names and cannot be overwritten; if they
+  already exist in your output directory, the model will crash.
+- Navigate to the `/ve_example/out/` folder.
+- Copy the `ve_full_model_configuration.toml` file and rename it, for example to
+  `HydroDefault_config.toml`.
+- In the new file, set `static = true` for all components
+  **except hydrology and core**, e.g.:
+
+  ```toml
+  [abiotic_simple]
+  static=true
+  ```
+
+Here is how you can do these steps programmatically:
+
+```{code-cell} ipython3
+:tags: [hide-input]
+
+from pathlib import Path
+import toml
+import copy
+
+# Create output directory
+output_dir = Path("/tmp/ve_example/HydroDefault_out/")
+output_dir.mkdir(parents=True, exist_ok=True)
+
+# Check that the directory is empty
+if output_dir.exists() and output_dir.is_dir():
+    for file in output_dir.iterdir():
+        if file.is_file():
+            file.unlink()
+
+# Load the original config
+config = toml.load("/tmp/ve_example/out/ve_full_model_configuration.toml")
```

Comment:
It is frustrating that we need two seperate TOML libraries to load and write TOML (and need to detect which version of a package we need depending on the system), but I don't think we should add `toml` to the requirements and use it. It hasn't been touched in two years and never made it to 1.0.